index.html

```diff
@@ -587,20 +637,25 @@ <h4>Script Type</h4>
                 <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Text</a> objects
                   in the <a>Target Recording Language</a>.
                   They MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
-                  in the case that the <a>Original</a> language is not the <a>Target Recording Language</a>, 
+                  in the case that their language is not the <a>Target Recording Language</a>, 
                   for context, to assist further processing.</p>
 
-                <aside class="note">The <a>Script Type</a> of a <a>DAPT Script</a> cannot necessarily be detected by inspecting the text content of the document.
-                For example, the adaptation of a <a>Translated Transcript</a> into a <a>Pre-recording Script</a>
-                can consist in replacing some words in the text content of a <a>Script Event</a> without altering the rest of the document.
-                In either case, <a>Text</a> objects that are translations have <a>Text Language Source</a> properties set to <a>Translation</a>.</aside>
+                <aside class="note">The <a>Script Type</a> of a <a>DAPT Script</a> cannot necessarily be detected
+                  by inspecting the text content of the document.
+                  For example, the adaptation of a <a>Translated Transcript</a> into a <a>Pre-recording Script</a>
+                  can consist in replacing some words in the text content of a <a>Script Event</a> without altering the rest of the document.
+                  In either case, <a>Translation</a> <a>Text</a> objects 
+                  have their <a>Text Language Source</a> properties
+                  set to the source language from which they were translated.</aside>
                 
-                <aside class="note">The <a>Text</a> objects in Audio Description <a>Script Events</a> are each expected to have
-                  a <a>Text Language Source</a> property of <a>Original</a> if their language is the <a>Primary Language</a>
-                  or, when representing in-image text,
-                  if they are in the same language as that text.
+                <aside class="note">The <a>Original</a> <a>Text</a> objects in Audio Description <a>Script Events</a>
+                  have an empty <a>Text Language Source</a> property
+                  if they represent visual elements of the scene that do not have an inherent language.
+                  Otherwise if they do represent visual elements with an inherent language, such as in-image text,
+                  they are required to have a <a>Text Language Source</a> that specifies a language.
```

Comment:
Can we use "required" in a NOTE?

Reply:
Yes we can. It is not considered "conformance language" as per this text in §6 Conformance:

>The key words MAY, MUST, MUST NOT, SHOULD, and SHOULD NOT in this document are to be interpreted as described in BCP 14 [RFC2119] [RFC8174] when, and only when, they appear in all capitals, as shown here.